library/ssl_srv.c

```diff
@@ -19,7 +19,7 @@
 
 #include "common.h"
 
-#if defined(MBEDTLS_SSL_SRV_C)
+#if defined(MBEDTLS_SSL_SRV_C) && defined(MBEDTLS_SSL_PROTO_TLS1_2)
```

Comment:
I understand that the whole file is guarded by `defined(MBEDTLS_SSL_PROTO_TLS1_2)`. Thus all the `defined(MBEDTLS_SSL_PROTO_TLS1_2)` guards inside the file are not necessary anymore and it would be better to remove them as we did in ssl_cli.c.

Reply:
strange. I remember I have remove them.